project/AppDependencies.scala

```diff
@@ -13,7 +13,7 @@ object AppDependencies {
     "org.typelevel"           %% "cats-core"                    % "2.10.0",
     "uk.gov.hmrc"             %% "internal-auth-client-play-28" % "1.8.0",
     "uk.gov.hmrc.objectstore" %% "object-store-client-play-28"  % "1.2.0",
```

Comment:
Can advance object-store-client and mockito version if you want with other suggestions

Reply:
https://github.com/hmrc/object-store-client/releases/tag/v1.3.0 object store is released for play 3.0 we are on 2.8 now and I wont upgrade, but can do mockito